plugins/search-react/api-report.md

```diff
@@ -454,7 +462,24 @@ export type SearchResultStateProps = SearchResultContextProps &
   Partial<SearchResultApiProps>;
 
 // @public (undocumented)
-export const SelectFilter: (props: SearchFilterComponentProps) => JSX.Element;
+export function SelectFilter<T extends JsonValue = string>(
+  props: SelectFilterProps<T>,
+): JSX.Element;
+
+// @public (undocumented)
+export interface SelectFilterProps<T extends JsonValue = string>
+  extends SearchFilterComponentProps {
+  // (undocumented)
+  emptyItemLabel?: string;
```

Comment:
What about renaming this property to be `allOptionText`?

Reply:
I don't mind, probably it's even more clear. So renamed